### PR TITLE
TH-1108 | Linkedevents v2 integration

### PR DESCRIPTION
**Pull request by Sartonon**
## Description :sparkles:

- Integrate linkedevents v2

## Additional notes :spiral_notepad:

### CODE REVIEW

**Review by nikomakela — APPROVED** [review]
Nice work!

File: src/__tests__/eventIntegration.test.ts [hunk 1]
```diff
@@ -40,10 +38,12 @@ it('resolves eventList correctly', async () => {
     data: [{ id: eventId, name: { fi: eventName } }],
     meta: { count: 1 },
   } as EventListResponse;
-  eventAPI.get = jest.fn().mockResolvedValue(mockData);
+  (eventAPI as any).get = jest.fn().mockResolvedValue(mockData);
 
   const res = await apolloTestServer.query({ query: EVENTS_QUERY });
 
+  if (res.errors) console.log(res.errors);
```
[the remark this answers is not in the document]
Comment (Sartonon) [reply src/__tests__/eventIntegration.test.ts:45]: Yeah that works but but I'm not sure if saving 2 characters is worth it here. I think it is little bit easier to understand when it says` if (res.errors)`

[the remark this answers is not in the document]
Comment (Sartonon) [reply src/__tests__/eventIntegration.test.ts:45]: But I'm fine with either :D

File: src/__tests__/eventIntegration.test.ts [hunk 2]
```diff
@@ -98,67 +107,169 @@ it('handles error correctly in eventsByIds', async () => {
...
+
+  expect(getMock).toHaveBeenCalledWith(
+    'event?event_type=Course&all_ongoing=true&all_ongoing_AND=asd&division=division1,division2' +
+      '&end=end&ends_after=09.10.2020&ends_before=10.10.2020&include=include&in_language=fi' +
+      '&is_free=true&keyword=keyword1,keyword2&keyword_AND=keywordAnd,keywordAnd2' +
+      '&keyword!=keywordNot&language=fi&location=location2,location3&page=10&page_size=10' +
+      '&publisher=publisher&sort=asc&start=10.10.2021&starts_after=10.20.2021&' +
+      'starts_before=10.10.2022&super_event=123aasd&super_event_type=course,event&text=testText' +
+      '&translation=translation'
+  );
```
[the remark this answers is not in the document]
Comment (Sartonon) [reply src/__tests__/eventIntegration.test.ts:200]: this is basically manually copy pasted "snapshot" test. I could just use inline snapshot here.

**Pull request merged**